ivy/data_classes/container/experimental/linear_algebra.py

```diff
@@ -1889,3 +1889,60 @@ def tucker(
             prune_unapplied=prune_unapplied,
             map_sequences=map_sequences,
         )
+
+    @staticmethod
+    def static_randomized_range_finder(
+        A: Union[ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        n_dims: Optional[int],
+        n_iter: Optional[int] = 2,
+        seed: Optional[int] = None,
```

Comment:
All of these arguments should have the `ivy.Container` type hint as well. Moreover, there are a few parameters missing in this - please take a look at some functions above for details on missing parameters.

Reply:
Also, please add the `out` argument.